### DISCUSSION TITLE
Einstellungen des Profils exportieren und speichern/Exporting and saving profile settings

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#423] [Q&A] [answered]
Hallo zusammen,

wir sitzen gerade mit 2 Leuten vor der App und finden nicht mehr die Funktion wo man sich seine angelegten Einstellungen exportieren kann. Wo war noch einmal der Knopf dafür? 

Hello everyone,

We are currently sitting in front of the app with 2 people and can no longer find the function where you can export the settings you have created. Where was the button for this again?

**Comment by devemux86** [comment, marked as answer]
Hi @ToToGSARider,

You can use the new profile screen to export / import profiles and transfer them to other devices. Please see the images.

Sie können den neuen Profilbildschirm verwenden, um Profile zu exportieren / importieren und auf andere Geräte zu übertragen.
Bitte sehen Sie die Bilder.

<img src="https://github.com/devemux86/cruiser/assets/3484020/fee36778-5daa-449f-a5e1-070146790cc2" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/1932f409-ea99-4161-9b7a-d5992b9bf1ed" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/6a64a3c7-eb43-496d-bf09-deaa0f3d4dfb" width="33%"/>

**Reply by ToToGSARider** [reply]
Dankeschön, genau diese Info hat uns gefehlt.

Thank you, this is exactly the information we were missing.
